### ISSUE TITLE
How can we use our library with Pixi

### ISSUE THREAD

**Issue by miguelmoraleda** [manuelodelain/sprite-anim#10]
PixiRenderer?

**Comment by manuelodelain** [comment]
maybe there is already an easy way to play a spritesheet animation in Pixi  ? 
Like http://www.goodboydigital.com/pixijs/docs/classes/MovieClip.html ? (I have never used Pixi).

**Comment by miguelmoraleda** [comment]
Yeah, there is an animation system. I can't remember why, but we had to create our own solution. Something was not cool about the PIXI's implementation. Probably we should digg more about it. I will ask my co-workers.
